### DISCUSSION TITLE
Webhook when configuration changes

### DISCUSSION THREAD

**Discussion by notscottsmith** [netbox-community/netbox#16521] [Help Wanted!]
Hi all,

I'm using the config-diff plugin for netbox and wanted to have a webhook fired off when a configuration is out of compliance. I'm not sure how to setup the event rules to fire off. I initially thought it would be attached to the device but I'm guessing something needs to be added to the plugin code to support the hook.

Has anyone done anything similar?

**Comment by jeremystretch** [comment]
What do you mean by "a configuration?" A configuration rendered by NetBox, or an operational config defined on a device?

**Reply by notscottsmith** [reply]
Well, both. The notification is meant to occur when there's a difference between the rendered config and the config on the device.
